packaging/generic-deb/debian/ecs.service

```diff
@@ -0,0 +1,31 @@
+# Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the
+# "License"). You may not use this file except in compliance
+# with the License. A copy of the License is located at
+#
+#     http://aws.amazon.com/apache2.0/
+#
+# or in the "license" file accompanying this file. This file is
+# distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR
+# CONDITIONS OF ANY KIND, either express or implied. See the
+# License for the specific language governing permissions and
+# limitations under the License.
+
+[Unit]
+Description=Amazon Elastic Container Service - container agent
+Documentation=https://github.com/aws/amazon-ecs-agent
+Requires=docker.service
+After=docker.service
+After=cloud-final.service
```

Comment:
the debian service doesnt wait for cloud-final.service in ecs-init. This is because this service usually wouldnt exist outside of AWS (for ecs anywhere), and so that when in AWS the ecs anywhere install script can be run in userdata.